juniper_official/interface/collect-logical-interface-stats.rule

```diff
@@ -0,0 +1,252 @@
+/*
+ * Collect Logical Interface traffic statistics for pathfinder dynamic topology.
+ *
+ * Req 60: https://paragon-automation.atlassian.net/browse/REQ-60
+ *
+ * Two inputs control detection
+ *
+ *   1) input-ifd-name, is a regular expression that matches the interfaces
+ *      that you would like to monitor.  By default it's '.*', which matches
+ *      all interfaces. Use something like 'ge.*' to match only gigabit
+ *      ethernet interfaces. 
+ *
+ *   2) input-ifl-no, is a regular expression that matches the IFL index number that
+ *      you would like to monitor.  By default it's '.*', which matches all
+ *      interfaces. Use something like '0-10*' to match only 0 to 10 IFLs.
+ *
+ */
+ healthbot {
```

Comment:
What is the scale that we are targeting for these rules?

Reply:
Response from Reinhard: For the scale numbers: Again, I haven't found any numbers. For Northstar customers it could be over 100k interfaces. So, I assume the requirement would be in that range.